hera_pspec/grouping.py

```diff
@@ -339,7 +346,14 @@ def average_spectra(uvp_in, blpair_groups=None, time_avg=False,
             except:
                 raise IndexError("blpair_weights must have the same shape as "
                                  "blpair_groups")
-    
+
+    stat_l = []
```

Comment:
can you add a brief comment here just describing what `stat_l` is and what this block of code is doing